### PR TITLE
Autogenerated/samsung.kies/5184e23a c758 4264 8581 d0f8993e25ce

### PR DESCRIPTION
**Pull request by upintheairsheep**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/106258)

### CODE REVIEW

**Comment by upintheairsheep** [comment]
It has some sort of issue with Microsoft.VCRedist.2008.x64 (x86) or similar.

**Comment by Trenly** [comment]
> It has some sort of issue with Microsoft.VCRedist.2008.x64 (x86) or similar.

It requires a very specific version of Microsoft.VCRedist.2008.x86 but it seems that old versions of VCRedist are giving me an error when trying to install

> Activation context generation failed for "C:\Users\WDAGUtilityAccount\AppData\Local\Temp\WinGet\Samsung.Kies.2.6.4.17113.1\KiesSetup.exe". Dependent Assembly Microsoft.VC90.MFC,processorArchitecture="x86",publicKeyToken="1fc8b3b9a1e18e3b",type="win32",version="9.0.21022.8" could not be found. Please use sxstrace.exe for detailed diagnosis.

**Comment by stephengillie** [comment]
Automatic Validation ended with:
InstallationVerification-x86-Scope_Undefined-Locale_Undefined-ErrorScreenshot_20230509031414

**Comment by stephengillie** [comment]
Manual Validation ended with:
> Activation context generation failed for "C:\Users\User\AppData\Local\Temp\WinGet\Samsung.Kies.2.6.4.17113.1\KiesSetup.exe". Dependent Assembly Microsoft.VC90.MFC,processorArchitecture="x86",publicKeyToken="1fc8b3b9a1e18e3b",type="win32",version="9.0.21022.8" could not be found. Please use sxstrace.exe for detailed diagnosis.

(Automated response - build 520.)

File: manifests/s/Samsung/Kies/2.6.4.17113.1/Samsung.Kies.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Samsung.Kies
+PackageVersion: 2.6.4.17113.1
```
Comment (stephengillie) [inline manifests/s/Samsung/Kies/2.6.4.17113.1/Samsung.Kies.installer.yaml:5]:
```suggestion
PackageVersion: 2.6.4.17113.1
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2008.x86
```

**Comment by stephengillie** [comment]
After adding the dependency, it appears that the install isn't unattended:
image

**Pull request closed without merging**